docs/Changelog.md

```diff
@@ -4984,6 +4984,50 @@ opset_import {
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Sort-1"></a>**Sort-1**</a>
+
+  Return a sorted copy of a tensor.
+
+#### Versioning
+
+This operator is used if you are using version 1 of the default ONNX operator set until the next BC-breaking change to this operator; e.g., it will be used if your protobuf has:
+
+~~~~
+opset_import {
+  version = 1
+}
+~~~~
+
+#### Attributes
+
+<dl>
+<dt><tt>axis</tt> : int</dt>
+<dd>Axis along which to sort</dd>
+<dt><tt>descending</tt> : int</dt>
```

Comment:
Why do we define bool attributes as `int`? Are there language bindings that don't support bool directly?